controller/Lists.php

```diff
@@ -475,45 +370,39 @@ public function rename()
         $data = ['renamed' => false];
 
         if ($this->hasRequestParameter('uri') && $this->hasRequestParameter('newName')) {
+            $listService = $this->getListService();
```

Comment:
We could have a `ListUpdater::updateByRequest(ServerRequest $request): ListUpdatedResponse` here to remove this logic from controller and make it testable.